src/dev/ci_setup/git_setup.sh

```diff
@@ -52,6 +52,9 @@ function checkout_sibling {
     }
 
     function pick_clone_target {
+      echo "To develop Kibana features against a specific branch of ${project} and being able to"
```

Comment:
Can we add some colors?

Reply:
Come'n Brandon, *colours*.  Lets be sensitive to our European counter parts who give words interesting spelling 😉

Reply:
Apologies... I guess now is as good a time as any to bring up adding automatic internationalization to PR comments to prevent embarrassing gaffes like this in the future.

Reply:
*internationali**s**ation*

Reply:
:rainbow: